### DISCUSSION TITLE
Voxel density values

### DISCUSSION THREAD

**Discussion by alessandroAM** [AcademySoftwareFoundation/openvdb#1316] [Q&A] [answered]
Hello folks, I recently started studying openVDB and I'm a bit confused. Is it possible to set the grid points (voxels) density values (driven from a greyscale 3D texture) so that I can basically represent a cloud? Thanks

**Comment by jmlait** [comment, marked as answer]
Yes.  Storing arbitrary grey-scale data in a sparse volume is a common use-case of VDB.

See the Moana cloud for an example.
https://media.disneyanimation.com/uploads/production/data_set_asset/1/asset/Cloud_Readme.pdf
